app/javascript/utils/distributions_and_transfers.js

```diff
@@ -106,3 +106,58 @@ $(function() {
     request_storage_location_and_populate_item(default_item);
   });
 });
+
+$(document).ready(function (e) {
+  $(document).on('change', '.dist_delivery_method', hide_show_shipping_cost);
+  show_shipping_cost();
+});
+
+function show_shipping_cost()
+{
+  var delivery_method_radio = $(".dist_delivery_method");
+  var shipping_cost_div = $(".distribution_shipping_cost");
+  var shipped = false;
+
+  if ( delivery_method_radio.length>0 && shipping_cost_div.length>0 ) {
+    $.each(delivery_method_radio, function(index, delivery_method) {
+       if(delivery_method.checked == true && delivery_method.value == "shipped")
+       {
+         shipped=true;
+       }
+    });
+
+    const shipping_cost_input = $("#dist_shipping_cost");
+    if (shipped) {
+      shipping_cost_input.removeClass("d-none");
+      shipping_cost_input.addClass("d-block");
+    }
+    else
+    {
+      shipping_cost_div.find("label").addClass("d-none")
```

Comment:
If all you're doing is hiding and showing a div, it's easier to do it with `toggle`:

`div.toggle(shipped)`

Reply:
here we are showing or hiding based on condition (delivery_method values), I guess toggle is used when there is no condition and same action is performed, wdyt?

Reply:
Nope, `toggle` can also be passed a boolean indicating whether to show or hide.